services/requester/requester.go

```diff
@@ -232,6 +238,41 @@ func (e *EVM) signAndSend(ctx context.Context, script []byte, args ...cadence.Va
 		return flow.EmptyID, fmt.Errorf("failed to send transaction: %w", err)
 	}
 
+	// get transaction status after it is submitted
+	go func(id flow.Identifier) {
+		const fetchInterval = time.Millisecond * 500
+		const fetchTimeout = time.Second * 60
+
+		fetchTicker := time.NewTicker(fetchInterval)
+		timeoutTicker := time.NewTicker(fetchTimeout)
+
+		defer fetchTicker.Stop()
+		defer timeoutTicker.Stop()
+
+		for {
+			select {
+			case <-fetchTicker.C:
+				res, err := e.client.GetTransactionResult(context.Background(), id)
+				if err != nil {
+					e.logger.Error().Err(err).Msg("failed to get transaction result")
```

Comment:
you should include the `id` in the message so its easier to debug

Reply:
ohh right